catalog/spatial/stanag4559/spatial-stanag4559-source/src/test/java/org/codice/ddf/spatial/ogc/stanag4559/catalog/source/server/MockSTANAG4559.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.ogc.stanag4559.catalog.source.server;
+
+import java.io.FileWriter;
+
+import org.apache.cxf.jaxrs.JAXRSServerFactoryBean;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.source.impl.LibraryImpl;
+import org.omg.CORBA.ORB;
+import org.omg.PortableServer.POA;
+import org.omg.PortableServer.POAHelper;
+
+public class MockSTANAG4559 {
+
+    public static void main(String args[]) throws Exception {
+        if(args.length != 1) {
+            System.out.println("Cannot run the mock server.  No ports are specified.");
+            return;
+        }
+
+        String[] ports = args[0].split(",");
+
+        int webPort = Integer.parseInt(ports[0]);
+
+        int corbaPort = Integer.parseInt(ports[1]);
+
+        ORB orb = runMockSTANAG4559(corbaPort);
+        runMockWebServer(webPort);
+
+        System.out.println("Server ready...");
+        orb.run();
+    }
```

Comment:
Per the ORB docs I think we need to call destroy before the process exits. Maybe add a shutdownHook to clean this up? https://docs.oracle.com/javase/7/docs/api/java/lang/Runtime.html#addShutdownHook(java.lang.Thread)